### ISSUE TITLE
Plugin registering with an incorrect plugin identifier

### ISSUE THREAD

**Issue by mtressl** [iharosi/homebridge-plex-v2#7]
Just found this in my startup log:

```
[4/28/2020, 9:17:41 AM] Loaded plugin: homebridge-plex-v2
[4/28/2020, 9:17:41 AM] Plugin 'homebridge-plex-v2' tried to register with an incorrect plugin identifier: 'homebridge-plex'. Please report this to the developer!
[4/28/2020, 9:17:41 AM] Registering accessory 'homebridge-plex-v2.Plex'
```
Doing as instructed!
Thanks for looking into it.

**Comment by iharosi** [comment]
Thanks!

**Comment by iharosi** [comment]
Fixed in e1cf9e8a095290e982efcb3d8d41217fd5b71aa7
